log process name and id

helunwencser opened the pull request:
Summary:
I am trying to debug my model running on vllm. I found that it is useful to log the process name and id.

This PR adds that.

Test Plan:
Saw the following in log:
`INFO 06-09 17:08:25 [utils.py:266] Started process EngineCore_0 with pid 1967179
`

Differential Revision: D76304325

houseroad reviewed (commented):
fix the linter?

houseroad commented:
Also could you provide the log examples in the test plan?

helunwencser commented:
Thanks for reviewing.

Fixed the linter.

Added test plan as well.

houseroad reviewed (commented):
CI output

```
diff --git a/vllm/v1/utils.py b/vllm/v1/utils.py
index 0669374..56a3a71 100644
--- a/vllm/v1/utils.py
+++ b/vllm/v1/utils.py
@@ -263,7 +263,8 @@ class CoreEngineProcManager:
         try:
             for proc in self.processes:
                 proc.start()
-                logger.info("Started process %s with pid %d", proc.name, proc.pid)
+                logger.info("Started process %s with pid %d", proc.name,
+                            proc.pid)
         finally:
             # Kill other procs if not all are running.
             if self.finished_procs():
```

Commit:
log process name and id (#19387)

Summary:
Pull Request resolved: https://github.com/vllm-project/vllm/pull/19387

I am trying to debug my model running on vllm. I found that it is useful to log the process name and id.

This PR adds that.

Reviewed By: houseroad, wushidonguc

Differential Revision: D76304325
